[Observability] Update alerts with relevant indices and permissions (#5493)
elastic/docs-content: solutions/observability/incident-management/create-an-error-count-threshold-rule.md

# Create an error count threshold rule [observability-create-error-count-threshold-alert-rule]

Create an error count threshold rule to alert you when the number of errors in a service exceeds a defined threshold. Threshold rules can be set at different levels: environment, service, transaction type, and/or transaction name.

:::{image} /solutions/images/serverless-alerts-create-rule-error-count.png
:alt: Create rule for error count threshold alert
:screenshot:
:::

## Requirements

> To create log threshold rules, you need the following:

Reviewer: ```suggestion
To create error count threshold rule, you need the following:
```

I think this was a typo?

> To create error count threshold rules", you need the following:

Reviewer 2: ```suggestion
To create error count threshold rules, you need the following:
```

> - {applies_to}`stack: ga` The permission for the [Infrastructure application](/solutions/observability/apm/secure-access-to-applications-ui.md).

Reviewer: The link points to the APM Application permissions, which is ok-ish.

The problem is: is it enough to grant a user the `Create an APM reader user` role to use the `Error count threshold`?

> - {applies_to}`stack: ga` The permission for the [Infrastructure application](/solutions/observability/apm/secure-access-to-applications-ui.md).

Reviewer: ```suggestion
- {applies_to}`stack: ga` The permission for the [Applications](/solutions/observability/apm/secure-access-to-applications-ui.md).
```

This is application, not infra

> - {applies_to}`stack: ga` The permission for the [Infrastructure application](/solutions/observability/apm/secure-access-to-applications-ui.md).

Reviewer 2: ```suggestion
- {applies_to}`stack: ga` The permission for the [APM application](/solutions/observability/apm/secure-access-to-applications-ui.md).
```

Error count threshold is an APM rule, not infra.
